Library/Homebrew/extend/fileutils.rb

```diff
@@ -132,6 +171,7 @@ def xcodebuild(*args)
   ensure
     ENV.update(removed)
   end
+
```

Comment:
I'd prefer if the PR didn't introduce empty lines unrelated to the changed code.